cpp/mg_utility/data_structures/graph_view.hpp

```diff
@@ -97,9 +98,18 @@ class GraphView {
   ///
   ///@param memgraph_id Memgraph's internal ID
   ///@return TSize
+  ///@throws mg_exception::InvalidIDException if vertex does not exist
   ///
   virtual TSize GetInnerNodeId(std::uint64_t memgraph_id) const = 0;
 
+  ///
+  ///@brief Get the Inner Node ID from Memgraph ID
+  ///
+  ///@param memgraph_id Memgraph's internal ID
+  ///@return std::optional<TSize>, which is null if vertex does not exist
+  ///
+  virtual std::optional<TSize> GetInnerNodeIdOpt(std::uint64_t memgraph_id) const = 0;
```

Comment:
Memgraph ID is GID and what is Inner Node ID?

Reply:
The inner node ID is specific to the `GraphView` data structure used by a few existing query modules. For a graph with `n` nodes, their inner IDs are 0...`n - 1`.

Reply:
Do we still need this function in the interface?

Reply:
We do – it’s used by older query modules (and their tests) that rely on `GraphView`.